src/main/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFsEngine.java

```diff
@@ -179,8 +179,11 @@ private VariantContext regenotypeVC(final VariantContext originalVC, final Refer
         // We could theoretically make 2 passes to re-create the genotypes, but that gets extremely expensive with large sample sizes.
         if (result.isPolymorphicInSamples()) {
             // For polymorphic sites we need to make sure e.g. the SB tag is sent to the annotation engine and then removed later.
-            final VariantContext reannotated = annotationEngine.annotateContext(result, features, ref, null, a -> true);
-            return new VariantContextBuilder(reannotated).genotypes(cleanupGenotypeAnnotations(reannotated, false)).make();
+            //final VariantContext reannotated = annotationEngine.annotateContext(result, features, ref, null, a -> true);
+            //return new VariantContextBuilder(reannotated).genotypes(cleanupGenotypeAnnotations(reannotated, false)).make();
+            VariantContext reannotated = new VariantContextBuilder(result).genotypes(cleanupGenotypeAnnotations(result, false)).make();
+            reannotated = annotationEngine.annotateContext(reannotated, features, ref, null, a -> true);
```

Comment:
you could reverse this change partly make the variable final and directly return the update. perhaps name the variable unannotated, preannotated?

Reply:
Also remove the commented out code.